components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/model/AccessTokenDO.java

```diff
@@ -79,6 +80,36 @@ public class AccessTokenDO extends CacheEntry {
 
     private int appResidentTenantId = MultitenantConstants.INVALID_TENANT_ID;
 
+    public String getAcr() {
```

Comment:
Remove these getters and setters and add new constructor to get values including these params and set them.